### ISSUE TITLE
tflint failing for gcp 2.0

### ISSUE THREAD

**Issue by gowthampatil** [terraform-linters/tflint#2595]
### Summary

Tflint check is failing with below error for PR 

Installing "google" plugin...
Panic: runtime error: invalid memory address or nil pointer dereference
 -> 0: main.main.func1: /main.go(25)
 -> 1: runtime.gopanic: /panic.go(860)
 -> 2: runtime.panicmem: /panic.go(336)
 -> 3: runtime.sigpanic: /signal_unix.go(931)
 -> 4: github.com/sigstore/sigstore-go/pkg/bundle.(*Bundle).TlogEntries: /bundle.go(300)
 -> 5: github.com/sigstore/sigstore-go/pkg/verify.VerifyTlogEntry: /tlog.go(40)
 -> 6: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).VerifyTransparencyLogInclusion: /signed_entity.go(809)
 -> 7: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).Verify: /signed_entity.go(606)
 -> 8: github.com/terraform-linters/tflint/plugin.(*SignatureChecker).VerifyAttestations: /signature.go(136)
 -> 9: github.com/terraform-linters/tflint/plugin.(*InstallConfig).Install: /install.go(134)
 -> 10: github.com/terraform-linters/tflint/cmd.(*CLI).init.func1: /init.go(49)
 -> 11: github.com/terraform-linters/tflint/cmd.(*CLI).withinChangedDir: /cli.go(200)
 -> 12: github.com/terraform-linters/tflint/cmd.(*CLI).init: /init.go(23)
 -> 13: github.com/terraform-linters/tflint/cmd.(*CLI).Run: /cli.go(108)
 -> 14: main.main: /main.go(38)
 -> 15: runtime.main: /proc.go(290)
 -> 16: runtime.goexit: /asm_amd64.s(1771)

TFLint crashed... :(
Please attach an output log, describe the situation and version that occurred and post an issue to https://github.com/terraform-linters/tflint/issues
Error: Process completed with exit code 1.

### Command

tflint

### TFLint Configuration

```HCL
Not sure from where i can get it
```

### Output

```Shell
Installing "google" plugin...
Panic: runtime error: invalid memory address or nil pointer dereference
 -> 0: main.main.func1: /main.go(25)
 -> 1: runtime.gopanic: /panic.go(860)
 -> 2: runtime.panicmem: /panic.go(336)
 -> 3: runtime.sigpanic: /signal_unix.go(931)
 -> 4: github.com/sigstore/sigstore-go/pkg/bundle.(*Bundle).TlogEntries: /bundle.go(300)
 -> 5: github.com/sigstore/sigstore-go/pkg/verify.VerifyTlogEntry: /tlog.go(40)
 -> 6: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).VerifyTransparencyLogInclusion: /signed_entity.go(809)
 -> 7: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).Verify: /signed_entity.go(606)
 -> 8: github.com/terraform-linters/tflint/plugin.(*SignatureChecker).VerifyAttestations: /signature.go(136)
 -> 9: github.com/terraform-linters/tflint/plugin.(*InstallConfig).Install: /install.go(134)
 -> 10: github.com/terraform-linters/tflint/cmd.(*CLI).init.func1: /init.go(49)
 -> 11: github.com/terraform-linters/tflint/cmd.(*CLI).withinChangedDir: /cli.go(200)
 -> 12: github.com/terraform-linters/tflint/cmd.(*CLI).init: /init.go(23)
 -> 13: github.com/terraform-linters/tflint/cmd.(*CLI).Run: /cli.go(108)
 -> 14: main.main: /main.go(38)
 -> 15: runtime.main: /proc.go(290)
 -> 16: runtime.goexit: /asm_amd64.s(1771)

TFLint crashed... :(
Please attach an output log, describe the situation and version that occurred and post an issue to https://github.com/terraform-linters/tflint/issues
Error: Process completed with exit code 1.
```

### TFLint Version

0.63.1

### Operating System

- [ ] Linux
- [ ] macOS
- [ ] Windows

**Comment by Poil** [comment]
Hi,

I have same issue with aws plugins since yesterday.

```
$ tflint --init
Installing "aws" plugin...
Panic: runtime error: invalid memory address or nil pointer dereference
 -> 0: main.main.func1: /main.go(25)
 -> 1: runtime.gopanic: /panic.go(860)
 -> 2: runtime.panicmem: /panic.go(336)
 -> 3: runtime.sigpanic: /signal_unix.go(931)
 -> 4: github.com/sigstore/sigstore-go/pkg/bundle.(*Bundle).TlogEntries: /bundle.go(300)
 -> 5: github.com/sigstore/sigstore-go/pkg/verify.VerifyTlogEntry: /tlog.go(40)
 -> 6: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).VerifyTransparencyLogInclusion: /signed_entity.go(809)
 -> 7: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).Verify: /signed_entity.go(606)
 -> 8: github.com/terraform-linters/tflint/plugin.(*SignatureChecker).VerifyAttestations: /signature.go(136)
 -> 9: github.com/terraform-linters/tflint/plugin.(*InstallConfig).Install: /install.go(134)
 -> 10: github.com/terraform-linters/tflint/cmd.(*CLI).init.func1: /init.go(49)
 -> 11: github.com/terraform-linters/tflint/cmd.(*CLI).withinChangedDir: /cli.go(200)
 -> 12: github.com/terraform-linters/tflint/cmd.(*CLI).init: /init.go(23)
 -> 13: github.com/terraform-linters/tflint/cmd.(*CLI).Run: /cli.go(108)
 -> 14: main.main: /main.go(38)
 -> 15: runtime.main: /proc.go(290)
 -> 16: runtime.goexit: /asm_amd64.s(1771)
TFLint crashed... :(
Please attach an output log, describe the situation and version that occurred and post an issue to https://github.com/terraform-linters/tflint/issues
```

I have to add `signature = "none"` to my configuration 
```
cat > ~/.tflint.hcl <<EOF
      plugin "terraform" {
        enabled = true
        preset  = "${TFLINT_PRESET}"
      }

      plugin "aws" {
        enabled = true
        version = "0.47.0"
        source  = "github.com/terraform-linters/tflint-ruleset-aws"
        # The pre-commit-terraform:latest image ships a tflint whose sigstore-go
        # crashes (nil pointer in Bundle.TlogEntries) while verifying the plugin
        # attestation during `tflint --init`. Skip signature verification until the
        # image ships a fixed tflint; the version is pinned and the source official.
        signature = "none"
      }

      config {
        force = false
        disabled_by_default = false

        varfile = [${TFLINT_VAR_FILE:+'"'${TFLINT_VAR_FILE}'"'}]
      }

      rule "terraform_standard_module_structure" {
         enabled = false
      }
EOF
```

Regards

**Comment by sramekmichal** [comment]
Hello, same problem here, but for AWS:
```
$ tflint -v
TFLint version 0.63.1
$ tflint --init
Installing "aws" plugin...
Panic: runtime error: invalid memory address or nil pointer dereference
 -> 0: main.main.func1: /main.go(25)
 -> 1: runtime.gopanic: /panic.go(860)
 -> 2: runtime.panicmem: /panic.go(336)
 -> 3: runtime.sigpanic: /signal_unix.go(931)
 -> 4: github.com/sigstore/sigstore-go/pkg/bundle.(*Bundle).TlogEntries: /bundle.go(300)
 -> 5: github.com/sigstore/sigstore-go/pkg/verify.VerifyTlogEntry: /tlog.go(40)
 -> 6: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).VerifyTransparencyLogInclusion: /signed_entity.go(809)
 -> 7: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).Verify: /signed_entity.go(606)
 -> 8: github.com/terraform-linters/tflint/plugin.(*SignatureChecker).VerifyAttestations: /signature.go(136)
 -> 9: github.com/terraform-linters/tflint/plugin.(*InstallConfig).Install: /install.go(134)
 -> 10: github.com/terraform-linters/tflint/cmd.(*CLI).init.func1: /init.go(49)
 -> 11: github.com/terraform-linters/tflint/cmd.(*CLI).withinChangedDir: /cli.go(200)
 -> 12: github.com/terraform-linters/tflint/cmd.(*CLI).init: /init.go(23)
 -> 13: github.com/terraform-linters/tflint/cmd.(*CLI).Run: /cli.go(108)
 -> 14: main.main: /main.go(38)
 -> 15: runtime.main: /proc.go(290)
 -> 16: runtime.goexit: /asm_amd64.s(1771)
TFLint crashed... :(
Please attach an output log, describe the situation and version that occurred and post an issue to https://github.com/terraform-linters/tflint/issues
```

**Comment by DanyloSafankov** [comment]
We have the same issue on `v0.61.0`

```
Installing "terraform" plugin...
Panic: runtime error: invalid memory address or nil pointer dereference
 -> 0: main.main.func1: /main.go(25)
 -> 1: runtime.gopanic: /panic.go(783)
 -> 2: runtime.panicmem: /panic.go(262)
 -> 3: runtime.sigpanic: /signal_unix.go(925)
 -> 4: github.com/sigstore/sigstore-go/pkg/bundle.(*Bundle).TlogEntries: /bundle.go(300)
 -> 5: github.com/sigstore/sigstore-go/pkg/verify.VerifyTlogEntry: /tlog.go(40)
 -> 6: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).VerifyTransparencyLogInclusion: /signed_entity.go(809)
 -> 7: github.com/sigstore/sigstore-go/pkg/verify.(*Verifier).Verify: /signed_entity.go(606)
 -> 8: github.com/terraform-linters/tflint/plugin.(*SignatureChecker).VerifyAttestations: /signature.go(136)
 -> 9: github.com/terraform-linters/tflint/plugin.(*InstallConfig).Install: /install.go(130)
 -> 10: github.com/terraform-linters/tflint/cmd.(*CLI).init.func1: /init.go(49)
 -> 11: github.com/terraform-linters/tflint/cmd.(*CLI).withinChangedDir: /cli.go(200)
 -> 12: github.com/terraform-linters/tflint/cmd.(*CLI).init: /init.go(23)
 -> 13: github.com/terraform-linters/tflint/cmd.(*CLI).Run: /cli.go(108)
 -> 14: main.main: /main.go(38)
 -> 15: runtime.main: /proc.go(285)
 -> 16: runtime.goexit: /asm_amd64.s(1693)
TFLint crashed... :(
Please attach an output log, describe the situation and version that occurred and post an issue to https://github.com/terraform-linters/tflint/issues
zwf-terraform-lint >>> Running the 'check' tflint mode for the directory: infrastructure/modules/datadog-production
Failed to initialize plugins; Plugin "aws" not found. Did you run "tflint --init"?
```
